extension/encoding/googlecloudlogentryencodingextension/internal/passthroughnlb/parser_test.go

```diff
@@ -0,0 +1,218 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package passthroughnlb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv "go.opentelemetry.io/otel/semconv/v1.34.0"
```

Comment:
Update to 1.38 to get rid of this error:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1b7307b97abdd4814a9f0afc5ab4e742ac634a99/.golangci.yml#L106